### PR TITLE
#1944 and #1792, French and Spanish translations

### PR DESCRIPTION
**Pull request by sanjuroj**

### CODE REVIEW

File: indicators/forms.py [hunk 1]
```diff
@@ -284,10 +284,12 @@ def get_helptext(disagg):
                 )
             return helptext
         self.fields['grouped_disaggregations'] = GroupedMultipleChoiceField(
+            # Translators:  disaggregation types that are available to all programs
```
Comment (axoplasm) [inline indicators/forms.py]: is this a *list* of disagg types available to all programs?

File: indicators/forms.py [hunk 2]
```diff
@@ -284,10 +284,12 @@ def get_helptext(disagg):
...
             [(_('Global disaggregations'),
               [(disagg.pk, str(disagg)) for disagg in global_disaggs],
               [get_helptext(disagg) for disagg in global_disaggs],
               [getattr(disagg, 'has_results') for disagg in global_disaggs])] +
+            # Translators:  disaggregation types that are available only to a specific country
```
Comment (axoplasm) [inline indicators/forms.py]: is this a _list_ of disagg types available only to a specific country?

File: indicators/indicator_plan.py [hunk 3]
```diff
@@ -27,7 +27,7 @@
 PERFORMANCE_INDICATOR = _('Performance Indicator')
 TARGETS = _('Targets')
 DATA_ACQUISITION = _('Data Acquisition')
-ANALYSES_AND_REPORTING = _('Analyses and Reporting')
```
Comment (axoplasm) [inline indicators/indicator_plan.py]: Why was this plural?

File: js/pages/tola_management_pages/country/components/edit_disaggregations.js [hunk 4]
```diff
@@ -326,7 +327,9 @@ class DisaggregationType extends React.Component {
                                     </label>
                                     <HelpPopover
                                         key={1}
+                                        // # Translators: Help text for the "selected by default" checkbox on the disaggregation form
                                         content={`<p>${gettext('When adding a program indicator, this disaggregation will be selected by default.  (It can be unselected for specific indicators)')}</p>
+                                            <!-- # Translators: Additional help text for the "selected by default" checkbox on the disaggregation form -->
```
Comment (axoplasm) [inline js/pages/tola_management_pages/country/components/edit_disaggregations.js]: is this the correct comment style?

File: templates/indicators/results/result_form_disaggregation_fields.html [hunk 5]
```diff
@@ -19,6 +19,7 @@
                 {% endif %}
             </a>
             <span class="formset-feedback invalid-feedback">
+                {# # Translators:  Allows the user to select only those items that need to be updated in some way#}
```
Comment (axoplasm) [inline templates/indicators/results/result_form_disaggregation_fields.html]: Do we need the second hash character for Translator notes in django templates?

File: indicators/forms.py (hunk 1 above)
Comment (sanjuroj) [reply indicators/forms.py]: @axoplasm Yep, both this and the one below are lists of disagg types.  Are you saying that adding the word "list" would be helpful?

File: indicators/indicator_plan.py (hunk 3 above)
Comment (sanjuroj) [reply indicators/indicator_plan.py]: No idea.  Marie asked to change it to the singular.

File: js/pages/tola_management_pages/country/components/edit_disaggregations.js (hunk 4 above)
Comment (sanjuroj) [reply js/pages/tola_management_pages/country/components/edit_disaggregations.js]: It seems to work, the comment appears in the .po file and doesn't appear on the page.

File: templates/indicators/results/result_form_disaggregation_fields.html (hunk 5 above)
Comment (sanjuroj) [reply templates/indicators/results/result_form_disaggregation_fields.html]: Ah, good catch, we don't need the second hash.

**Pull request merged**